assets/src/blocks/Media/MediaFrontend.js

```diff
@@ -0,0 +1,61 @@
+import { MediaEmbedPreview } from './MediaEmbedPreview';
+import { MediaElementVideo } from './MediaElementVideo';
+import { useMediaDataFetch } from './useMediaDataFetch';
+
+const { __ } = wp.i18n;
+
+const wrapAndRemoveSize = embedHtml => {
+  const wrapperDiv = document.createElement('div');
+  wrapperDiv.innerHTML = embedHtml;
+  if (wrapperDiv.firstChild && wrapperDiv.firstChild.src && wrapperDiv.firstChild.src.indexOf('youtube') > -1) {
+    wrapperDiv.className = 'embed-container';
+    wrapperDiv.firstChild.removeAttribute('width');
+    wrapperDiv.firstChild.removeAttribute('height');
+    return wrapperDiv.outerHTML;
+  } else {
```

Comment:
Return early here. We could add a rule to enforce that https://eslint.org/docs/rules/no-else-return

Reply:
Probably better to invert the if, so that we only need to put `return embedHtml` inside it, instead of the preceding 4 lines.

Reply:
Ok, NOW it's 1000% done, double-checked. Resolving.